client/src/components/Snackbar/index.jsx

```diff
@@ -1,17 +1,25 @@
 /* eslint-disable react/require-default-props */
 import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import CheckCircleIcon from '@material-ui/icons/CheckCircle';
-import ErrorIcon from '@material-ui/icons/Error';
-import InfoIcon from '@material-ui/icons/Info';
-import CloseIcon from '@material-ui/icons/Close';
-import { amber, green } from '@material-ui/core/colors';
-import IconButton from '@material-ui/core/IconButton';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton } from '@material-ui/core';
 import Snackbar from '@material-ui/core/Snackbar';
 import SnackbarContent from '@material-ui/core/SnackbarContent';
-import WarningIcon from '@material-ui/icons/Warning';
-import { makeStyles } from '@material-ui/core/styles';
+import {
+  Error as ErrorIcon,
+  Info as InfoIcon,
+  Close as CloseIcon,
+  CheckCircle as CheckCircleIcon,
+  Warning as WarningIcon,
+} from '@material-ui/icons';
+import { amber, green } from '@material-ui/core/colors';
+
+import SNACKBAR_ACTIONS from '../../store/actions/snackbarActions';
+
+const { SET_SNACKBAR_OPEN } = SNACKBAR_ACTIONS;
 
```

Comment:
would `import {SET_SNACKBAR_OPEN} from "../../store/actions/snackbarActions';` be a simpler idea?

Reply:
It doesn't work!! trust me I tried.

Reply:
It's to do with how you're exporting the module

```
const OPEN_SNACKBAR = 'OPEN_SNACKBAR';

export default { OPEN_SNACKBAR };
```

Try using `export const ...` to export each action, or use `export { ACTION_ONE, ACTION_TWO }` without the `default`

https://exploringjs.com/impatient-js/ch_modules.html#overview-syntax-of-ecmascript-modules

Reply:
Fixed.